app/hotels/src/map/HotelSwipeItem.js

```diff
@@ -60,28 +63,28 @@ export const HotelSwipeItem = ({ width, data: hotel }: Props) => {
   const review = idx(hotel, _ => _.hotel.review);
 
   return (
-    <View style={[{ width }]}>
+    <View style={[styles.container, { width }]}>
+      <View>
+        <NetworkImage
+          style={styles.image}
+          source={{
+            uri: image,
+          }}
+          resizeMode="contain"
+        />
+      </View>
       <View style={styles.description}>
-        <View>
-          <NetworkImage
-            style={styles.image}
-            source={{
-              uri: image,
-            }}
-            resizeMode="contain"
-          />
-        </View>
-        <View>
-          <Text style={styles.hotelName}>{name}</Text>
-          {getReview(review)}
-          <Text style={styles.price}>
-            {price &&
-              price.currency &&
-              price.amount && (
-                <Price currency={price.currency} amount={price.amount} />
-              )}
-          </Text>
-        </View>
+        <Text style={styles.hotelName} numberOfLines={1}>
```

Comment:
Oh wow, I didn't know about `numberOfLines` property... 👍 

@VaclavSir used `react-native-read-more-text` in single hotel detail. It's basically a wrapper around this property with additional features. But this is definitely good enough.